src/components/templates/Camp/components/Map.scss

```diff
@@ -44,7 +44,6 @@ $border-radius: 28px;
       display: block;
     }
     .playa-venue-img {
-      box-shadow: 0 8px 20px 0 rgba(0, 0, 0, 0.2);
```

Comment:
whoops these are unrelated fixes for camp venue where it had some unnneeded shadows.